commerce-subscription-web/src/main/resources/META-INF/resources/subscription_entry/details.jsp

```diff
@@ -119,11 +111,7 @@ if (maxSubscriptionCycles > 0) {
 									return true;
 								}
 
-								if (parseInt(val, 10) > 0) {
-									return true;
-								}
-
-								return false;
+								return (parseInt(val, 10) > 0);
```

Comment:
The way we had it was actually the kosher way at Liferay :/. This was is functionally the same, and more concise, but the other way is a little more obvious, so that's the preferred way.